DetectorDescription/OfflineDBLoader/BuildFile.xml

```diff
@@ -1,4 +1,11 @@
 <use name="DetectorDescription/Core"/>
+<use name="DetectorDescription/DDCMS"/>
+<use name="dd4hep"/>
+<use name="dd4hep-core"/>
+<use name="dd4hep-geant4"/>
+<use name="geant4core"/>
```

Comment:
@cvuosalo - Introducing G4 dependency adds an extra layer of conversion. That means we will not have the same geometry description between the first XML parsing from the files and the second XML parsing from the DB payload.

Reply:
The G4 dependency is a left-over. It will be removed soon.